starter-code/src/components/ProductBox.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+
+class ProductBox extends Component {
+  
+  constructor(props){
+    super(props);
+    this.state = { quantity: 0}
+  }
+
+
+
+  onChangeQuantity = (event) => {
+    if (event.target.value >= 0){
+      this.setState({quantity: event.target.value})
```

Comment:
Aquí debería ser `this.setState({ quantity: this.state.quantity + event.target.value })`. Al hacerlo como lo tienes, solo añade alimentos cuando la cantidad es diferente a la añadida anteriormente. Si quisiera añadir de uno en uno no podría porque igualando al valor en vez de devolver la que sería el resultado de la suma pisa el valor anterior.